book/src/design/nullifiers.md

```diff
@@ -29,22 +30,26 @@ We care about several security properties for our nullifiers:
 - **Balance:** can I forge money?
 
 - **Note Privacy:** can I gain information about notes only from the public block chain?
+
```

Comment:
These spaces separating different bullet-point levels are necessary for valid reStructuredText, not Markdown.

Reply:
Ah, good point. I always get confused, so I usually write in a subset that minimises the differences.